### PR TITLE
CHANGELOG update

### PR DESCRIPTION
**Pull request by hannahramadan**
CHANGELOG update for community member-submitted PR #2475

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 Version <dev> hardens the browser agent insertion logic to better proactively anticipate errors.
 
+- **Feature: Store tracer state on thread-level**
+
+  A new configuration option, `thread_local_tracer_state`, stores New Relic's tracer state on the thread-level, as opposed to the default fiber-level storage. This configuration is turned off by default. Thanks to community member [@markiz](https://github.com/markiz) who contributed the idea, code, configuration option, and tests for this new feature! [PR#2475](https://github.com/newrelic/newrelic-ruby-agent/pull/2475).
```
Comment (kaylareopelle) [inline CHANGELOG.md:9]:
```suggestion
  A new configuration option, `thread_local_tracer_state`, stores New Relic's tracer state on the thread-level, as opposed to the default fiber-level storage. This configuration is turned off by default. Our thanks go to community member [@markiz](https://github.com/markiz) who contributed the idea, code, configuration option, and tests for this new feature! [PR#2475](https://github.com/newrelic/newrelic-ruby-agent/pull/2475).
```

Comment (kaylareopelle) [reply CHANGELOG.md:9]: Did you intend to leave this entry out of the version summary on line 5?

**Review by kaylareopelle — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hannahramadan) [reply CHANGELOG.md:9]: Oh I did not! I'll update—great catch.

**Review by kaylareopelle — APPROVED** [review]
Thanks, Hannah!

**Pull request merged**